include/deal.II/base/geometry_info.h

```diff
@@ -4425,7 +4426,7 @@ GeometryInfo<2>::child_cell_on_face(const RefinementCase<2> &ref_case,
   // this cell adjacent to the subface of a possibly
   // refined neighbor. this simplifies setting neighbor
   // information in execute_refinement.
-  const unsigned int
+  constexpr unsigned int
     subcells[2][RefinementCase<2>::isotropic_refinement][faces_per_cell]
             [max_children_per_face] = {
```

Comment:
This is the change that causes the compiler error. Just remove the changes to `geometry_info.h` from this patch -- they are unrelated to what you want to achieve here anyway, so they should go into separate pull requests.

Reply:
Oh, this is what I add to avoid the test error. I have to check it by eyes since my compiler didn't complain anything. Can I just leave this if it pass the test? I'm afraid any commit would break this again 😢

Reply:
Well, for some reason it fails the Continuous Integration checks, and so we cannot merge it. It is also unrelated. If you want to make this change, please make it in a separate pull request and take it out of this one.